graph/min_cost_b_flow/info.toml

```
@@ -0,0 +1,40 @@
+title = 'Minimum cost b-flow'
+timelimit = 5.0
```

```suggestion
timelimit = 10.0
```

We usually set TL=10s to the problem that is expected many types solutions. (We don't focus on giving "TLE" into slow solution :) )